### PR TITLE
Remove duplicate words

### PR DESCRIPTION
**Pull request by nschonni**
Playing around with https://github.com/errata-ai/vale to find duplicate words.
Ran `vale --glob='*.md' .` with the following `.vale`
```
# This goes in a file named either `.vale` or `_vale`.

StylesPath = path/to/some/directory
MinAlertLevel = error # suggestion, warning or error

[*.{md,txt}] # Only Markdown and .txt files
# List of styles to load
BasedOnStyles = vale, Joblint
# Style.Rule = {YES, NO, suggestion, warning, error} to
# enable/disable a rule or change its level.
vale.Editorializing = NO
```
Some false positives for:
- build. EX: "Team Build build definition"
- Iteration

I split the commits out, but if it's too noisy you can cherry-pick what you want

### CODE REVIEW

**Comment by nschonni** [comment]
Unfortunatly this is running into the line ending/encoding issues in the repo again. Looking at https://github.com/MicrosoftDocs/vsts-docs/pull/1048/files?utf8=%E2%9C%93&diff=unified&w=1 you'll see 98 lines changed

File: docs/notifications/collaborate-in-a-team-room.md [hunk 1]
```diff
@@ -1,157 +1,158 @@
...
+A team room is created for [each team that gets created](../work/scale/multiple-teams.md). Team administrators can create additional rooms and manage their team rooms.  
+
+Members of the Project Administrators groups can create and administer team rooms that they have created. And, members of the Project Collection Administrators groups can create and administer all team rooms.  
+
+## Join the conversation
+1. From your team's home page, open the room.  
+
+	![Team room tile on team project home page](_img/ALM_CT_TeamroomTile.png) 
+	
+	If you don't see the room tile, [(VSTS) ask your account administrator to grant you a Basic license](../accounts/add-account-users-assign-access-levels.md) or [(on-premises TFS) request Basic access from your TFS administrator](../security/change-access-levels.md).
```
Comment (nschonni) [inline docs/notifications/collaborate-in-a-team-room.md]: Problem with the rebase, will fix

File: docs/report/dashboards/configure-burndown-burnup-widgets.md [hunk 2]
```diff
@@ -1,281 +1,282 @@
...
+* Projected completion date, based on historical burndown and scope increase rates
+
+<!---
+
+## Configure the widget
+### Pre-requisites
+
+In order to configure the Burndown and Burndown widgets, you must have the following in place:
+
+* Installed the [Analytics Marketplace extension](https://marketplace.visualstudio.com/items?itemName=ms.vss-analytics). You must be an account owner or a member of the [Project Collection Administrator group](../../security/set-project-collection-level-permissions.md) to add extensions.
```
Comment (nschonni) [inline docs/report/dashboards/configure-burndown-burnup-widgets.md]: Bad rebase, will fix

File: docs/report/dashboards/velocity-chart-data-store.md [hunk 3]
```diff
@@ -1,91 +1,92 @@
...
+*	Determine how your team wants to [treat bugs](../../work/customize/show-bugs-on-backlog.md). If your team chooses to treat bugs like requirements, bugs will show up on the backlog and be counted within the Velocity chart and forecasting. 
+*	[Set your team's area path](../../work/customize/set-area-paths.md). The forecast tool will forecast those items based on your team's default settings. These settings can specify to include items in area paths under the team's default or exclude them.     
+*	Don't  create a hierarchy of backlog items and bugs. The Kanban board, sprint backlog, and task board only show the last node in a hierarchy, called the leaf node. For example, if you link items within a hierarchy that is four levels deep, only the items at the fourth level appear on the Kanban board, sprint backlog, and task board. <br/>Instead of nesting requirements, bugs, and tasks, we recommend that you maintain a flat list��only creating parent-child links one level deep between items. Use [Features to group requirements or user stories](../../work/backlogs/organize-backlog.md). You can quickly map stories to features, which creates parent-child links in the background.
```
Comment (nschonni) [inline docs/report/dashboards/velocity-chart-data-store.md]: Bad encoding, will fix

File: docs/report/extend-analytics/odata-query-guidelines.md [hunk 4]
```diff
@@ -322,7 +322,7 @@ To resolve this problem, use the OData batch endpoint as explained in the specif
 
 We restrict use of the batch endpoint from handling a batch of multiple requests. A single request can still have only one query. If you try to send a batch of several queries, the operation will fail with the following error message. The only solution is to split queries into multiple requests.
 
-> *The Analytics Service doesn��t support processing of multiple operations which the current batch message contains. The Analytics Service uses OData batch in order to support POST requests, but requires you limit the operation to a single request.*
+> *The Analytics Service doesn��t support processing of multiple operations which the current batch message contains. The Analytics Service uses OData batch in order to support POST requests, but requires you limit the operation to a single request.*
```
Comment (nschonni) [inline docs/report/extend-analytics/odata-query-guidelines.md]: Bad encoding, will fix

File: docs/work/customize/reference/use-categories-to-group-work-item-types.md [hunk 5]
```diff
@@ -1,110 +1,111 @@
...
-Most of these categories are self-explanatory, and most only contain one WIT within the category. The exception to this rule is the Hidden Types Category.  
-  
-If you have created WITs that act in similar ways and you want to treat them in similar ways as those defined by the above categories, then you will want to add them to the category. For example, if you have defined one or more types of bugs, then you might want to add those types to the Bug Category. In this way, the process configuration will automatically treat these bug types as they do the standard bug WIT. Or, you can customize the Requirement Category to include two or more WITs which will then appear on the product backlog.  
-  
-<a name="hiddentypes"></a> 
-###Hidden Types category  
- The Hidden Types Category specifies the set of WITs that you do not want users to create manually. By default this set includes:  
-  
--   [Code Review Request and Code Review Response](../../../tfvc/day-life-alm-developer-suspend-work-fix-bug-conduct-code-review.md )    
--   [Feedback Request and Feedback Response](../../../project/feedback/get-feedback.md)
```
Comment (nschonni) [inline docs/work/customize/reference/use-categories-to-group-work-item-types.md]: Bad rebase, will fix

File: docs/work/customize/update-customized-process-template.md [hunk 6]
```diff
@@ -1,508 +1,509 @@
...
+	</td>
+	<td>
+	[Develop code and manage pending changes](../../tfvc/develop-code-manage-pending-changes.md) from the My Work page in Team Explorer. My Work only appears when you have Visual Studio Premium or Visual Studio Ultimate installed. 
+	</td>
+	</tr>
+	<tr>
+	<td>
+	Feedback
+	</td>
+	<td>[Request feedback](../../feedback/get-feedback.md). <br/>
```
Comment (nschonni) [inline docs/work/customize/update-customized-process-template.md]: Bad rebase, will fix

File: docs/work/scale/review-team-plans.md [hunk 7]
```diff
@@ -1,205 +1,206 @@
...
+> - How to add field criteria, customize cards, and add markers
+> - How to open a plan from the list of defined plans 
+> - How to work with the interactive elements of plans and change the plan view
+> - What permissions are required to create and view plans  
+
+
+## Prerequisites
+In order to add and configure a plan, you must have the following in place:  
+- [Installed the Plans extension](#install-plans)  
+- Be granted [Basic access or greater](../../security/access-levels.md) (Users with Stakeholder access can't add or view plans)
```
Comment (nschonni) [inline docs/work/scale/review-team-plans.md]: Bad rebase, will fix

File: docs/work/scale/review-team-plans.md [hunk 8]
```diff
@@ -1,205 +1,206 @@
...
+
+## Prerequisites
+In order to add and configure a plan, you must have the following in place:  
+- [Installed the Plans extension](#install-plans)  
+- Be granted [Basic access or greater](../../security/access-levels.md) (Users with Stakeholder access can't add or view plans)  
+- [Defined iteration paths](../customize/set-area-paths.md) for the team project  
+- [Configured teams](multiple-teams.md) and [set team defaults and team sprints](set-team-defaults.md)   
+- Teams have defined [user stories](../backlogs/create-your-backlog.md), features, or other product or portfolio backlogs and assigned them to iterations  
+
+All users with [basic access](../../security/change-access-levels.md) can view, add, and configure Delivery Plans. (Accounts assigned to [Stakeholder access](../../security/get-started-stakeholder.md), however, don't have access to Delivery Plans.)
```
Comment (nschonni) [inline docs/work/scale/review-team-plans.md]: Bad rebase, will fix

File: docs/work/scale/review-team-plans.md [hunk 9]
```diff
@@ -1,205 +1,206 @@
...
+
+Once you have [installed Delivery Plans](#install-plans), the Plans page appears under the Work hub.
+
+<img src="_img/review-team-plans-open-plans.png" alt="Plans menu item" style="border: 1px solid #C3C3C3;" /> 
+
+0. To add a plan, click New Plan from the Plans page. 
+
+	<img src="_img/review-team-plans-no-plans-defined.png" alt="Create new plan" style="border: 1px solid #C3C3C3;" /> 
+
+	All users, except users [assigned Stakeholder access](../../security/change-access-levels.md), have permissions to create a plan and manage the plans they create. To manage permissions for a plan, see [Set permissions and access for work tracking, Manage or edit Delivery Plans](../../security/set-permissions-access-work-tracking.md#plan-permissions).
```
Comment (nschonni) [inline docs/work/scale/review-team-plans.md]: Bad rebase, will fix

**Comment by nschonni** [comment]
@KathrynEE I've rebase this and resolved the conflicts. You can see the non-encoding related changes here https://github.com/MicrosoftDocs/vsts-docs/pull/1048/files?utf8=%E2%9C%93&diff=unified&w=1

**Comment by steved0x** [comment]
@nschonni I'm merging this, thank you for your contribution.

**Pull request merged**